src/test/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -0,0 +1,224 @@
+      const dcr = createDCRBaseForAcceptance()
+      dcr.courtDecisionType = DecisionType.DEFENDANT
+      dcr.formaliseRepaymentPlan = new FormaliseRepaymentPlan(new FormaliseRepaymentPlanOption('xyz', 'xyz'))
+      const errMsg = 'Unknown formalise repayment option xyz'
+      expect(() => converter.covertToClaimantResponse(dcr)).to.throw(Error, errMsg)
+    })
+
+    it(' Accept court decision with ccj', () => {
+      expect(converter.covertToClaimantResponse(createDCRWithCourtDecisionType(intentionOfImmediatePayment,DecisionType.COURT,FormaliseRepaymentPlanOption.REQUEST_COUNTY_COURT_JUDGEMENT))).to.deep.eq(
+        {
+          'type': 'acceptation',
+          'amountPaid': 10,
+          'claimantPaymentIntention': {
+            'paymentOption': 'IMMEDIATELY'
+          },
+          'courtDetermination': {
+            ...courtDecisionInstalments,
+            'disposableIncome': 200
+          },
+          'formaliseOption': 'CCJ',
+          'determinationDecisionType': 'COURT'
+        })
+    })
+
+    it(' Accept court decision favouring defendant payment intent ', () => {
+      expect(converter.covertToClaimantResponse(createDCRWithCourtDecisionType(intentionOfPaymentInFullBySetDate,DecisionType.DEFENDANT,FormaliseRepaymentPlanOption.REQUEST_COUNTY_COURT_JUDGEMENT))).to.deep.eq(
+        {
+          'type': 'acceptation',
+          'amountPaid': 10,
+          'claimantPaymentIntention': {
+            'paymentOption': 'BY_SPECIFIED_DATE',
+            'paymentDate': new LocalDate(2018,12,31).toMoment()
+          },
+          'courtDetermination': {
+            ...courtDecisionInstalments,
+            'disposableIncome': 200
+          },
+          'formaliseOption': 'CCJ',
+          'determinationDecisionType': 'DEFENDANT'
+        })
+    })
+
+    it(' Accept court decision favouring claimant payment intent ', () => {
+      expect(converter.covertToClaimantResponse(createDCRWithCourtDecisionType(intentionOfPaymentByInstallments,DecisionType.CLAIMANT,FormaliseRepaymentPlanOption.REQUEST_COUNTY_COURT_JUDGEMENT))).to.deep.eq(
+        {
+          'type': 'acceptation',
+          'amountPaid': 10,
+          'claimantPaymentIntention': {
+            'paymentOption': 'INSTALMENTS',
+            'repaymentPlan': {
+              'firstPaymentDate': new LocalDate(2018,12,31).toMoment(),
+              'instalmentAmount': 100,
+              'paymentSchedule': 'EVERY_MONTH'
+            }
+          },
+          'courtDetermination': {
+            ...courtDecisionInstalments,
+            'disposableIncome': 200
+          },
+          'formaliseOption': 'CCJ',
+          'determinationDecisionType': 'CLAIMANT'
+        })
+    })
+
+    it(' Reject court decision and refer to judge ', () => {
+      const dcr = createDCRWithCourtDecisionType(intentionOfImmediatePayment,DecisionType.COURT,FormaliseRepaymentPlanOption.REQUEST_COUNTY_COURT_JUDGEMENT)
```

Comment:
plz give a meaningful name to the const here

Reply:
dcr stands for **D**raft**C**laimant**R**esponse

Reply:
please change to `draftClaimantResponse` and add type as well